openqa-investigate

```diff
@@ -232,32 +233,41 @@ fetch-investigation-results() {
             job=$(client-get-job-state "$other_id")
             state=$(echo "$job" | runjq -r '.state') || return $?
             # at least one job is not finished, come back later
-            [[ $state != 'done' ]] && return 142
+            is-finished "$state" || return 142
 
             result=$(echo "$job" | runjq -r '.result') || return $?
             echo "$investigate_type|$other_id|$result"
         fi
     done
 }
 
+is-finished() {
+    [[ $1 == done || $1 == cancelled ]]
```

Comment:
From the failed style check:
```
 In openqa-investigate line 245:
    [[ $1 == done || $1 == cancelled ]]
             ^-- SC1010 (warning): Use semicolon or linefeed before 'done' (or quote to make it literal).
```

Reply:
Maybe using " to prevent this, also in the other checks